operator/cmd/mesh/shared.go

```diff
@@ -95,41 +95,26 @@ func confirm(msg string, writer io.Writer) bool {
 	return false
 }
 
-// K8sConfig creates a rest.Config, Clientset and controller runtime Client from the given kubeconfig path and context.
-func K8sConfig(kubeConfigPath string, context string) (*rest.Config, *kubernetes.Clientset, client.Client, error) {
-	restConfig, clientset, err := InitK8SRestClient(kubeConfigPath, context)
+func KubernetesClients(kubeConfigPath, context string, l clog.Logger) (kube.ExtendedClient, client.Client, error) {
+	rc, err := kube.DefaultRestConfig(kubeConfigPath, context, func(config *rest.Config) {
+		config.QPS = 200
```

Comment:
Does the comment below still apply?

```golang
	// We are running a one-off command locally, so we don't need to worry too much about rate limitting
	// Bumping this up greatly decreases install time
```

Reply:
Oops, copy+pasted from somewhere else. nice catch. Will revert numbers back and add the comment back